### PR TITLE
examples: fix rewrite example

### PR DESCRIPTION
**Pull request by jan--f**
Seems like the resource-attributes example was largely just copied and
the description did not match the config fragments.

Signed-off-by: Jan Fajerski <jfajersk@redhat.com>

### CODE REVIEW

File: examples/rewrites/README.md [hunk 1]
```diff
@@ -4,7 +4,15 @@
 
 RBAC differentiates in two types, that need to be authorized, resources and non-resources. A resource request authorization, could for example be, that a requesting entity needs to be authorized to perform the `get` action on a particular Kubernetes Deployment.
 
-Take the following example. We want to deploy a [prometheus-example-app](https://github.com/brancz/prometheus-example-app), and protect it with the kube-rbac-proxy. In this example we require a requesting entity to be allowed to call the `proxy` subresource on a Kubernetes Service called `kube-rbac-proxy`. This is configured in the file passed to the kube-rbac-proxy with the `--config-file` flag. Additionally the `--upstream` flag has to be set to configure the application that should be proxied to on successful authentication as well as authorization.
+Take the following example. We want to deploy a
```
Comment (brancz) [inline examples/rewrites/README.md:7]: We don't wrap content in readmes, I don't feel strongly about changing that, but if we do we should do it in a separate PR and consistently everywhere.

**Review by brancz — COMMENTED** [review]
Could you for future us explain the changes in either the PR description or the commit? Thank you!

File: examples/rewrites/README.md (hunk 1 above)
Comment (jan--f) [reply examples/rewrites/README.md]: I figured in a rendered README it makes little difference but :+1: for consistency.

**Comment by brancz** [comment]
E2e tests seem to have failed. Is that a flake @s-urbaniak ? :thinking:

**Comment by s-urbaniak** [comment]
@brancz yes they seem to be flaky, i am investigating :thinking:

**Comment by s-urbaniak** [comment]
fwiw the e2e flake is tracked in https://github.com/brancz/kube-rbac-proxy/issues/108

**Comment by s-urbaniak** [comment]
@jan--f do you mind to rebase? the flakes should be fixed by now :crossed_fingers:

**Comment by s-urbaniak** [comment]
lgtm, thank you!
